settings.php

```diff
@@ -37,5 +37,26 @@
             new lang_string('duplicatemaxactivities_description', 'block_massaction'),
             5
         ));
+
+        $pluginmanager = \core_plugin_manager::instance();
+        $plugins = [];
+        foreach (array_keys($pluginmanager->get_installed_plugins('format')) as $plugin) {
+            $plugins[$plugin] = new lang_string('pluginname', 'format_' . $plugin);
+        }
+        // Sort alphabetically. Custom sort function needed, because lang_string is an object.
+        uasort($plugins, function($a, $b) {
+            return strcmp($a->out(), $b->out());
+        });
+
+        // These are the formats supported by the maintainer.
+        $supportedformatsbydefault = ['weeks' => 1, 'topics' => 1, 'topcoll' => 1];
```

Comment:
We should add the currently supported plugins in 3.9 to 3.11, right?
https://github.com/Syxton/moodle-block_massaction/blob/a3c388a722672c9ca494ad31354b73495ddccec7/block_massaction.php#L51-L53